### PR TITLE
Free tier sample apps

### LINKED ISSUE #5738: CC: Adapt CRDB app tutorials for CC
From Cal Hacks event report: "The switch between CRDB and CC docs is frustrating and confusing. Users (hackers and our engineers) try to use CRDB dev tutorials for CC and get stuck on the certs part. I need to prioritize CC-specific and language-specific dev tutorials post-beta."

### PR DESCRIPTION
**Pull request by lnhsingh**
Updates to the prioritized sample apps, which will include:

- [x] Python - psycopg2
- [ ] Python - SQLAlchemy
- [ ] Python - Django
- [ ] Java - Hibernate
- [ ] Java - JDBC
- [ ] Go - pgx
- [ ] Go - pq
- [ ] Go - GORM

Closes #5738.

### CODE REVIEW

**Comment by lnhsingh** [comment]
@ianjevans / @ericharmeling - adding you as my reviewers for this. Let me know if I should add anybody else (PM or Eng?)

File: _includes/v20.2/app/create-a-database.md [hunk 1]
```diff
@@ -48,12 +45,11 @@
     > CREATE DATABASE bank;
     ~~~
 
-1. Give the user the necessary permissions:
+<!-- 1. Give the your user the necessary permissions:
```
Comment (ianjevans) [inline _includes/v20.2/app/create-a-database.md:48]:
```suggestion
<!-- 1. Give your user the necessary permissions:
```

**Review by ianjevans — CHANGES_REQUESTED** [review]

**Review by ianjevans — APPROVED** [review]
I didn't notice at first that the suggested change was in a commented out block. Would be good to fix anyway, but proceed.

**Review by ericharmeling — COMMENTED** [review]
Looks great! Just a couple comments. 

We also should probably update the help string in the python repo:
https://github.com/cockroachlabs/hello-world-python-psycopg2/blob/master/example.py#L159

(That help text actually had CC stuff, but was removed in anticipation of this update: https://github.com/cockroachlabs/hello-world-python-psycopg2/commit/a8d63dd151c45a734555ac69477d9f02e2b353f5)

> Reviewed 4 of 4 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9339#-:-MRPoNPE-NSqIR3hUhbI:b9k92zn)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @lnhsingh)

___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 90 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRPmSNN5HYgn4cN9gbI:-MRPmSNN5HYgn4cN9gbJ:bq2tpnp) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L90)):*
> ```
> that you copied from the [earlier](#set-up-your-cluster-connection) on the **Connection info** modal
> ```

that you copied [earlier](#set-up-your-cluster-connection) from the **Connection info** modal
___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 97 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRPnt2ABH27mqeT_Qhl:-MRPnt2ABH27mqeT_Qhm:bv0jv25) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L97)):*
<details><summary><i alt="summary">Quoted 5 lines of code…</i></summary>

> ```
> {% include copy-clipboard.html %}
> ~~~ shell
> $ python3 example.py \
> "postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster-name>.bank?sslmode=verify-full&sslrootcert=<your_certs_directory>/cc-ca.crt"
> ~~~
> ```
</details>


I'd move this to before the `Before running the code, update the connection string...` bit (and logically after the `The Python code is a command-line utility that accepts the connection string to CockroachDB as a command-line argument.` bit).

Just a suggestion though :)

**Review by lnhsingh — COMMENTED** [review]
TFTRs! If this looks good to you all, you can get started on updating the other priority apps, and push to this branch (if that's easiest? I'm also fine merging this and then you all creating your own branches). I'm going to start working on the other Free Tier docs, but can circle back around to help with the remaining Sample Apps when I have time.

Also opened a PR to update the python help text: https://github.com/cockroachlabs/hello-world-python-psycopg2/pull/4

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9339#-:-MRRvC5D-HCskrQ9MQ1z:bgv45xk)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling and @ianjevans)

___
*[_includes/v20.2/app/create-a-database.md, line 48 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MR6_DQuRbuMihbYB95k-r1-48:-MRRs0N8BHs7SVZ7nin-:b-c0oki3) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/_includes/v20.2/app/create-a-database.md#L48)):*
<details><summary><i>Previously, ianjevans (Ian Evans) wrote…</i></summary><blockquote>

```suggestion
<!-- 1. Give your user the necessary permissions:
```
</blockquote></details>

Deleted this - not needed now that we use the default user that is created
___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 90 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRPmSNN5HYgn4cN9gbI:-MRRsJUFDuJjpTaRrAp8:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L90)):*
<details><summary><i>Previously, ericharmeling (Eric Harmeling) wrote…</i></summary><blockquote>

> ```
> that you copied from the [earlier](#set-up-your-cluster-connection) on the **Connection info** modal
> ```

that you copied [earlier](#set-up-your-cluster-connection) from the **Connection info** modal
</blockquote></details>

Done.
___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 97 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRPnt2ABH27mqeT_Qhl:-MRRs_CV5bxijs6vBddO:b-i5gpsz) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L97)):*
<details><summary><i>Previously, ericharmeling (Eric Harmeling) wrote…</i></summary><blockquote>

> ```
> {% include copy-clipboard.html %}
> ~~~ shell
> $ python3 example.py \
> "postgres://<username>:<password>@free-tier.gcp-us-central1.cockroachlabs.cloud:26257/<cluster-name>.bank?sslmode=verify-full&sslrootcert=<your_certs_directory>/cc-ca.crt"
> ~~~
> ```

I'd move this to before the `Before running the code, update the connection string...` bit (and logically after the `The Python code is a command-line utility that accepts the connection string to CockroachDB as a command-line argument.` bit).

Just a suggestion though :)
</blockquote></details>

Yeah that makes sense- now all of the "replace x" and "this will be pre-populated" info is all together.

**Review by ericharmeling — APPROVED** [review]
LGTM after a quick link fix. Thanks Lauren!

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9339#-:-MRV9RMb3IjvBSk9a3Sh:behvsnh)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling, @ianjevans, and @lnhsingh)

___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 92 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRV8RMVDROyF6gATn7k:-MRV8RMVDROyF6gATn7l:b-n4bvne) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L92)):*
> ```
> #create-a-database
> ```

This is breaking the build. I think it needs to be `#step-3-create-a-database`?

**Review by lnhsingh — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/docs/9339)* status: :shipit: complete! 0 of 0 LGTMs obtained (waiting on @ericharmeling and @ianjevans)

___
*[v20.2/build-a-python-app-with-cockroachdb.md, line 92 at r1](https://reviewable.io/reviews/cockroachdb/docs/9339#-MRV8RMVDROyF6gATn7k:-MRVEjTwCkq3WcDvc5ls:b-896fix) ([raw file](https://github.com/cockroachdb/docs/blob/35dd93afa4d55542e2afbbce7b58abe8e2dafd4a/v20.2/build-a-python-app-with-cockroachdb.md#L92)):*
<details><summary><i>Previously, ericharmeling (Eric Harmeling) wrote…</i></summary><blockquote>

> ```
> #create-a-database
> ```

This is breaking the build. I think it needs to be `#step-3-create-a-database`?
</blockquote></details>

Done.

**Pull request merged**